tests/compile_time.rs

```diff
@@ -0,0 +1,41 @@
+use std::time::{Duration, Instant};
+
+use xshell::{cmd, Shell};
+
+#[test]
+fn fixed_cost_compile_times() {
+    let sh = Shell::new().unwrap();
+
+    let _p = sh.push_dir("cbench");
+    let baseline = compile_bench(&sh, "baseline");
+    // FIXME: Don't have Interent rn, can't compile duct :-(
+    // let _ducted = compile_bench(&sh, "ducted");
```

Comment:
```suggestion
    // FIXME: Don't have internet rn, can't compile duct :-(
    // let _ducted = compile_bench(&sh, "ducted");
```

also, if you pushed this up, then you must have internet, so maybe you can rm this fixme? (:

Reply:
Indeed!